### ISSUE TITLE
[Request] Adult Flag improvement

### ISSUE THREAD

**Issue by BitPyrate** [HDInnovations/UNIT3D-Community-Edition#3679]
Adult Flag should be an optional option on the upload page.
Self declaration is good where TMDB Fails to flag smut.

**Comment by Roardom** [comment]
I would recommend editing tmdb itself, that way when users download the content to their plex/jellyfin/emby/kodi/whatever, users will be able to hide the content from their family-friendly libraries as well.

**Comment by BitPyrate** [comment]
Yes but many tiltles are not availible on tmdb. Be it onlyfans or other content there should be a method a person can seld declare. Updating tmdb is nice but we should focus on the accoricy of our own dbs first.
